### PR TITLE
Separated the sheetId based on the country the application sent to

### PR DESCRIPTION
**Pull request by SarKurd**
This PR needs further tests
Solve #111

### CODE REVIEW

File: api/src/data/update-sheet.js [hunk 1]
```diff
@@ -69,7 +67,8 @@ function saveApplicant(
         emailSent,
         assignmentFileUrl,
         assignmentUrl,
-        assignmentMessage
+        assignmentMessage,
+        country
```
Comment (M3kH) [inline api/src/data/update-sheet.js]: - [] This require to update the spreadsheet

File: api/src/utils/dev-config.js [hunk 2]
```diff
@@ -54,7 +54,15 @@ function getConfig() {
     if (!process.env.DEVELOPMENT)
         return {
             // This needs to be converted to an environment variable
-            spreadSheetId: "1CRF9TbwSayVq9yhrUCCv038DG0jPGBN7ehOyZa4mvE0"
+            nl: {
+                spreadSheetId: "1CRF9TbwSayVq9yhrUCCv038DG0jPGBN7ehOyZa4mvE0"
+            },
+            dk: {
+                spreadSheetId: "1CRF9TbwSayVq9yhrUCCv038DG0jPGBN7ehOyZa4mvE0"
```
Comment (M3kH) [inline api/src/utils/dev-config.js]: @NoerGitKat @wouterkleijn  could you guys provide 2 additional `spreadSheetId`?

**Comment by M3kH** [comment]
So far this looks good. We need additional `spreadSheetId`.

**Comment by SarKurd** [comment]
No merge === close :D

**Pull request closed without merging**